core/loop.gms

```diff
@@ -9,21 +9,21 @@
 *--------------------------------------------------------------------------
 ***         solveoptions
 *--------------------------------------------------------------------------
-option limcol    = 0;
```

Comment:
this was about how many lines of equations are displayed, right? 
will removing it have any effect on the debug runs that are automatically started after 2 infes? (just making sure we don't get full.lst files that are suddenly 20 times as large as before :-)